raiden/ui/web/src/app/services/raiden.config.ts

```diff
@@ -14,6 +15,7 @@ interface RDNConfig {
     http_timeout?: number;
     settle_timeout?: number;
     reveal_timeout?: number;
+    network_type: number;
```

Comment:
Please, make it optional and a string: `network_type?: string`, defaulting below to test

Reply:
Or better, if you change the enum to be a string enum, you can use the enum type directly in the interface I think